src/System.Runtime/src/System.Runtime.csproj

```diff
@@ -38,6 +38,10 @@
     <Compile Include="System\Reflection\RuntimeReflectionExtensions.cs" />
     <Compile Include="System\Runtime\CompilerServices\Attributes.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == '' OR '$(TargetGroup)' == 'netstandard1.7' OR '$(TargetGroup)' == 'uap101aot'">
```

Comment:
You shouldn't need `'$(TargetGroup)' == 'netstandard1.7'` condition, as that will never happen.

---
In reply to: [82454050](https://github.com/dotnet/corefx/pull/12392#discussion_r82454050) [](ancestors = 82454050)